index.html

```diff
@@ -11,24 +11,27 @@
     <section class="section-nav">
       <header>
         <!--Navigation Menu-->
+
         <nav class="nav">
-          <a class="logo-text">Jamila Moseka</a>
-          <img
-            class="menu-bar menu-hover"
-            src="images/menu1.png"
-            alt="menu options"
-          />
-          <ul class="nav-menu">
-            <li><a class="nav-links" href="#">Portfolio </a></li>
-            <li><a class="nav-links" href="#">About</a></li>
-            <li><a class="nav-links" href="#">Contact</a></li>
-            <li>
-              <img class="email-icon" src="images/Union.png" alt="email icon" />
-            </li>
-          </ul>
+            <a class="logo-text">Jamila Moseka</a>
+            <div id="id-menu-bar" class="menu-bar">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+            <ul class="primary-nav">
+              <li><a class="nav-links" href="#">Portfolio</a></li>
+              <li><a class="nav-links" href="#">About</a></li>
+              <li><a class="nav-links" href="#">Contact</a></li>
```

Comment:
- [ ] Please make sure that when the user clicks any of the mobile menu options, a correct part of the page is displayed. To do that you have to put the proper part of the project inside `href=""` eg:
`<li><a href="#portfolio">Portfolio</a></li>`
this link will take to  any section with `portfolio` id: `<section id="portfolio"> `

- [ ] Finally, make sure that when the user clicks any option of the mobile menu, the mobile menu disappear.

Reply:
Resolved. Respective links were added to the nav lists. 
The mobile menu disappears upon clicking the humberger button.